views/js/qtiRunner/core/QtiRunner.js

```diff
@@ -290,12 +290,18 @@ define([
     };
 
     QtiRunner.prototype.showModalFeedback = function(modalFeedback, callback){
-
+        var $modalFeedback;
         if(modalFeedback instanceof ModalFeedback){
             //load (potential) new qti classes used in the modal feedback (e.g. math, img)
             this.renderer.load(function(){
-                //render the modal feedback
-                $('#modalFeedbacks').append(modalFeedback.render());
+                $modalFeedback = $('#modalFeedbacks').find('#' + modalFeedback.getSerial());
```

Comment:
Could you store `$('#modalFeedbacks')` into a variable to prevent doing the query twice when it's appended.